app/lib/infrastructure/tts_repository.dart

```diff
@@ -0,0 +1,114 @@
+part of 'package:word_link/domain/controllers/tts_controller.dart';
+
+class _TtsRepository extends TtsController {
+  final FlutterTts _flutterTts = FlutterTts();
```

Comment:
For the fuctions it is good practice to set it in the initialize so that it will set once and also include all of the parameters.
So set the instance to late.

Call it like this in the initialize method
` _flutterTts = FlutterTts()..setLanguage('en-US')..setSpeechRate(0.5)` and so on.
Usually all of the settings should be part of the contractor and not added up like I have shown you, check if you can fit all of the values in the constructor of FlutterTts.

Reply:
?
Move FlutterTts() to the initialize function